charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,1580 @@ spec:
+                                                  type: array
+                                                  items:
+                                                    type: string
+                                      topologyKey:
+                                        type: string
+                        config:
+                          type: object
+                          properties:
+                            jvm:
+                              type: object
+                              properties:
+                                G1HeapRegionSize:
+                                  type: string
+                                concGCThreads:
+                                  type: string
+                                extraFlags:
+                                  type: array
+                                initialRAMPercentage:
+                                  type: string
+                                initiatingHeapOccupancyPercent:
+                                  type: string
+                                maxCachedBufferSize:
+                                  type: string
+                                maxDirectMemorySize:
+                                  type: string
+                                maxGcPauseMillis:
+                                  type: string
+                                maxRAMPercentage:
+                                  type: string
+                                minRAMPercentage:
+                                  type: string
+                                parallelGCThreads:
+                                  type: string
+                                permSize:
+                                  type: string
+                                reservedCodeCacheSize:
+                                  type: string
+                            logLevel:
+                              type: string
+                            taskMaxWorkerThreads:
+                              type: string
+                            taskMinDrivers:
+                              type: string
+                        nodeSelector:
+                          type: object
+                        resources:
+                          type: object
+                          properties:
+                            limits:
+                              type: object
+                              properties:
+                                cpu:
+                                  type: string
+                                memory:
+                                  type: string
+                            requests:
+                              properties:
+                                cpu:
+                                  type: string
+                                memory:
+                                  type: string
+                        terminationGracePeriodSeconds:
+                          type: integer
+                        tolerations:
```

Comment:
Tolerations could steal from https://github.com/coreos/prometheus-operator/blob/master/example/prometheus-operator-crd/prometheus.crd.yaml#L3413 for more of the structure